File: 7kyu/(7 kyu) Build a square/(7 kyu) Build a square.ts
export function generateShape(int: number): string {
  let string = '';
  for (let i = 0; i < int * int; i++) {
    if (i % int === 0 && i !== 0) {
      string += '\n';
    }
    string += '+';
  }
  return string;
}

File: 7kyu/(7 kyu) Build a square/(7 kyu) Build a square.rs
// #1
// fn generate_shape(n: i32) -> String {
//     let mut str = vec![];
//     for _ in 0..n {
//         str.push("+".repeat(n as usize));
//     }
//     str.join("\n")
// }
// #2
// fn generate_shape(n: i32) -> String {
//     vec!["+".repeat(n as usize); n as usize].join("\n")
// }
// #3
fn generate_shape(n: usize) -> String {
    vec!["+".repeat(n); n].join("\n")
}
